src/Nox.Solution/Validation/Domain/EntityPersistenceValidator.cs

```diff
@@ -13,6 +16,17 @@ public EntityPersistenceValidator(string entityName)
             RuleFor(ep => ep.TableName)
                 .NotEmpty()
                 .WithMessage(ep => string.Format(ValidationResources.EntityPersistenceTableNameEmpty, entityName));
+
+            RuleFor(ep => ep.IsAudited)
+                .NotEqual(true)
+                .When(ep => IsOwnedByAnyOtherEntity(entityName, entities))
```

Comment:
the entity has a IsOwnedEntityproperty, that you can use without this linq

Reply:
please update from main, thks

Reply:
Updated. Thanks for the info.